idtoken/validate_test.go

```diff
@@ -27,18 +28,28 @@ const (
 	testAudience = "test-audience"
 )
 
+var (
+	expiry int64 = 233431200
+	// one second before the token expires
+	beforeExp = func() time.Time { return time.Unix(233431199, 0) }
+	// one second after the token expires
+	afterExp = func() time.Time { return time.Unix(233431201, 0) }
```

Comment:
Make this a const, then reference it in `beforeExp` and `afterExp`?

Reply:
I like it, done.